mlflow/fastai/__init__.py

```diff
@@ -422,6 +422,7 @@ def load_model(model_uri, dst_path=None):
 @autologging_integration(FLAVOR_NAME)
 def autolog(
     log_models=True,
+    log_datasets=True,
```

Comment:
Do we need to add this argument to `mlflow.fastai.autolog()`, `mlflow.pytorch.autolog()`, etc? I think that `mlflow.autolog()` is a bit clever and checks to see which arguments are defined by the framework-specific autologging methods using `inspect`, so we don't need to define unused args here. @prithvikannan can you confirm that's the case and, if so, remove the `log_datasets` arg from flavors that don't implement dataset autologging?

Reply:
Turns out we have to have this in there. This is the same pattern as other args like `silent` which are not used in the code for fastai.